Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -22,6 +23,17 @@ public class TapToPlace : MonoBehaviour, IInputClickHandler
         [Tooltip("Supply a friendly name for the anchor as the key name for the WorldAnchorStore.")]
         public string SavedAnchorFriendlyName = "SavedAnchorFriendlyName";
 
+        [Tooltip("Place parent on tap instead of current game object.")]
+        public bool PlaceParentOnTap = false;
+
+        [Tooltip("Specify the parent game object to be moved on tap, if the immediate parent is not desired.")]
+        public GameObject ParentGameObjectToPlace;
+
+        /// <summary>
+        /// Keeps track of if the user is moving the object or not.
+        /// </summary>
+        public bool IsBeingDragged;
```

Comment:
@HodgsonSDAS I'm thinking about how it will be used as a base class. For instance:

`public override Update() {`
` if (IsBeingPlaced) { ... }`
`base.Update();`
`}`

Or if it's in an outside class accessing this component:

`if (gameObject.GetComponent<TapToPlace>().IsBeingDragged) { }`
vs
`if (gameObject.GetComponent<TapToPlace>().placing) { }`